framework/src/parser/Parser.C

```diff
@@ -521,10 +635,12 @@ Parser::parse(const std::string & input_filename)
   hit::EnvEvaler env;
   hit::ReplaceEvaler repl;
   FuncParseEvaler fparse_ev;
+  UnitsConversionEvaler units_ev;
   hit::BraceExpander exw(_input_filename);
   exw.registerEvaler("raw", raw);
   exw.registerEvaler("env", env);
   exw.registerEvaler("fparse", fparse_ev);
+  exw.registerEvaler("units", units_ev);
```

Comment:
Should this maybe go after the `replace` evaler?

Reply:
On second thought - this probably doesn't matter - since evalers are all executed in a single sweep over the AST - rather than one sweep per evaler.

Reply:
To be clear - over the brace expression AST inside every parameter - and they are evaluated depth-first.